client/slack.go

```diff
@@ -208,3 +222,41 @@ func newUser(u *slack.User) user {
 		LastName:  u.Profile.LastName,
 	}
 }
+
+func newReactionEvent(e *slack.ReactionAddedEvent, u, itemUser *slack.User) reactionEvent {
+	return reactionEvent{
+		Type:     e.Type,
+		User:     newUser(u),
+		ItemUser: newUser(itemUser),
+		Item: ReactionItem{
+			Type:      e.Item.Type,
+			Channel:   e.Item.Channel,
+			Timestamp: e.Item.Timestamp,
+		},
+		Reaction:       e.Reaction,
+		EventTimestamp: e.EventTimestamp,
+	}
+}
+
+type ReactionItem struct {
```

Comment:
```suggestion
type reactionItem struct {
```

as per original suggestion